src/components/StatesButton/StatesButton.st.css

```diff
@@ -19,7 +19,7 @@
 .root::content {
     line-height: 1.5;
     display: block;
-    margin: 3px;
+    padding: 3px;
```

Comment:
@jonathanadler I've replaced margin with padding because the CSS here: https://github.com/wix/wix-ui-tpa/blob/master/src/components/Button/Button.st.css#L56 is overriding this CSS. In general, I don't like replacing them, but here it feels ok because it is on the content element and there's nothing between the content and the root element.